BH/D2Stubs.cpp

```diff
@@ -75,19 +75,29 @@ DWORD __declspec(naked) __fastcall D2CLIENT_ClickParty_ASM(RosterUnit* RosterUni
 	}
 }
 
-void __declspec(naked) __fastcall D2CLIENT_PlaySound(int SoundNo)
+__declspec(naked) int __stdcall D2CLIENT_PlaySound_STUB(int nSound, UnitAny* pUnit, int nTriggerFrame, int param_4, int param_5)
 {
 	__asm
 	{
-		push ebx
-		mov ebx, ecx
-		push 0
-		push 0
-		push 0
-		push 0
-		call D2CLIENT_PlaySound_I
-		pop ebx;
-		ret
+		push EBX
+		mov EBX, [esp + 8]                                   // nSound
+		push[esp + 24]                                      // param_5
+		push[esp + 24]                                      // param_4
+		push[esp + 24]                                      // nTriggerFrame
+		push[esp + 24]                                      // pUnit
+		call D2CLIENT_PlaySound                              // 0x88A70
+		pop EBX
+		retn 20
+	}
+}
+
+__declspec(naked) void __stdcall D2CLIENT_PlayItemDropSounds_STUB(UnitAny* pUnit)
```

Comment:
What is this for? Or just leaving it in case we want to use it later?

Reply:
It's used down below in `GetItemFromPacket_NewGround`
https://github.com/Project-Diablo-2/BH/pull/25/files#diff-69d30ea176935d54c7fc5af24d86ff1f471bd5084156a999490aa7dbf922d361R402

It's actually `D2CLIENT_PlaySound_STUB` above that's unused (for now at least). And yeah, originally thought I'd need it but ended up not. It already existed I just updated it to match the pd2 repo.